config/locales/en/errors.yml

```diff
@@ -189,21 +189,21 @@ en:
         steps/case/is_client_remanded_form:
           attributes:
             is_client_remanded:
-              inclusion: Select yes if your client has the custody remanded
+              inclusion: Select yes if a court has remanded your client in custody
             date_client_remanded:
-              blank: Date client custody was remanded cannot be blank
-              blank_day: Date client custody was remanded must include a day
-              blank_month: Date client custody was remanded must include a month
-              blank_year: Date client custody was remanded must include a year
-              blank_day_month: Date client custody was remanded must include a day and month
-              blank_day_year: Date client custody was remanded must include a day and year
-              blank_month_year: Date client custody was remanded must include a month and year
+              blank: Enter when they were remanded
+              blank_day: Date they were remanded must include a day
+              blank_month: Date they were remanded must include a month
+              blank_year: Date they were remanded must include a year
+              blank_day_month: Date they were remanded must include a day and month
+              blank_day_year: Date they were remanded must include a day and year
+              blank_month_year: Date they were remanded must include a month and year
               invalid: Enter a valid date
```

Comment:
@anyone looking - Ripan and I discussed and agreed that this and the following 4 errors were more specific than what is in the spreadsheet so left them alone. I am confirming with Chris O